pages/_error.tsx

```diff
@@ -0,0 +1,52 @@
+// CSS
+import styles from '../styles/Error.module.scss';
+// Components
+import NavBar from '../components/NavBar';
+import Button, { ButtonType } from '../components/Button';
+
+export default function ErrorPage({ 
+  statusCode 
+}:{
+  statusCode: any
```

Comment:
Woah why is this an `any`?

can you also break the type definiton into a seperate interface like:
```ts
interface Props {
  statusCode: any; // replace any with whatever type it is meant to be, probably `number` but it might say on the next docs
}
```

This just makes the code more readable.

Reply:
I noticed below in the body you do `statusCode ? '${statusCode}' : 'default err'`, so maybe it should be `statusCode?: any` but again replace `any` with whatever its meant to be.